### PR TITLE
Explorer patch ( mongoose + gulpify + small cleanup )

### PR DESCRIPTION
**Pull request by btenterprise2020**
- [x] Updated mongoose version ( https://github.com/btenterprise2020/etc-explorer/pull/16 ).
- [x] Applied gulp for easier management ( https://github.com/btenterprise2020/etc-explorer/pull/18 ).
- [x] Minor cleanup ( https://github.com/btenterprise2020/etc-explorer/pull/21 ).
- [x] Document improvement.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -6,33 +6,31 @@
...
+  2. Go to the explorer subdirectory.
+  `cd explorer`
 
-`npm install`
+  3. Set up default configurations.
+  `cp config.example.json config.json`
 
-Install mongodb:
+  4. Install Node.js dependencies.
+  `npm install`
```
Comment (phyro) [inline README.md]: just curious, why not use the `docker-compose up` ?

File: config.example.json [hunk 2]
```diff
@@ -12,14 +12,14 @@
         "symbol": "ETC",
         "name": "Ethereum Classic",
         "title": "Ethereum Classic Block Explorer",
-        "author": "Elaine, Cody, Hackmod, Bakon",
```
Comment (phyro) [inline config.example.json]: I think we should ask authors if they are ok with this change
cc @elaineo @realcodywburns @hackmod not sure what handle Bakon has on github

File: docker-compose.yml [hunk 3]
```diff
@@ -26,13 +26,5 @@ services:
       MONGO_URI: 'mongodb://db/explorerDB'
     depends_on:
       - db
-  richlist:
```
Comment (phyro) [inline docker-compose.yml]: This just removes the richlist from the docker-compose right? is the idea that people now just run the `richlist.js` script?

File: README.md (hunk 1 above)
Comment (btenterprise2020) [reply README.md]: Well let's leave docker as an option 😂 ( I am still using local installation without docker deployment )

File: config.example.json (hunk 2 above)
Comment (btenterprise2020) [reply config.example.json]: Actually, that is not related to copyright changes nor the authors, it is just an additional html tag with the author section, but they are not required by google search engine anymore.

File: docker-compose.yml (hunk 3 above)
Comment (btenterprise2020) [reply docker-compose.yml]: 'tools/richlist.js is optional and it wouldn't be needed if user finishes full sync with 'tools/sync.js'. It is only needed when user want to add more accounts on database.  Example: https://github.com/ethereumclassic/explorer/pull/289.

**Comment by btenterprise2020** [comment]
Issue https://github.com/ethereumclassic/explorer/issues/310 fixed on https://github.com/ethereumclassic/explorer/pull/305/commits/2715d12b94eabeafc300219674aaa7cf92f469c5

**Pull request merged**